include/psa/crypto_config.h

```diff
@@ -935,6 +935,26 @@
  * This section sets PSA specific settings.
  * \{
  */
+
+/**
+ * \def MBEDTLS_CTR_DRBG_C
+ *
+ * Enable the CTR_DRBG AES-based random generator.
+ * The CTR_DRBG generator uses AES-256 by default.
+ * To use AES-128 instead, set #MBEDTLS_PSA_CRYPTO_RNG_STRENGTH to 128.
+ *
+ * AES support can either be achieved through built-in AES or PSA. Built-in is
+ * the default option when present otherwise PSA is used.
+ *
+ * Module:  drivers/builtin/src/ctr_drbg.c
+ *
+ * Requires: MBEDTLS_PSA_CRYPTO_C, PSA_WANT_KEY_TYPE_AES and
+ *           PSA_WANT_ALG_ECB_NO_PADDING
+ *
+ * This module provides the CTR_DRBG AES random number generator.
+ */
+#define MBEDTLS_CTR_DRBG_C
```

Comment:
I am a bit confused as to why we are adding configuration options in a commit that is labeled 'Remove legacy crypto options'

Not a blocker but the commit message is a bit confusing without a body